### ISSUE TITLE
Downloader can't find any courses - KIT

### ISSUE THREAD

**Issue by freedome97** [DeOldSax/iliasDownloaderTool#66]
The Karlsruher Institute of Technology has updatet their Illias from Version 5 to 7.
Ever since, the IlliasDownloaderTool is not able refresh or even find any courses.
Here an image to demonstrate what I mean:
![image](https://user-images.githubusercontent.com/50106400/148850738-413b71a0-6801-4ec8-986e-bf73b8f92a85.png)

Any ideas how to resolve this problem?
Greetings

**Comment by DeOldSax** [comment]
@freedome97 i guess the html structure changed slightly or maybe the rendering process (more javascript involved?). I don't have an account anymore for the KIT Ilias -> so i can only guess. If someone feels like solving the issue i am happy to point to the most likely relevant files.

**Comment by freedome97** [comment]
@DeOldSax Yeah the homepage changed pretty much. So below is a screen comparing the new (left) with the old (right) Illias
The login page you will be redirected to after clicking the marked buttons stayed the same.
![gitkit](https://user-images.githubusercontent.com/50106400/149123697-71669228-a3b0-49d9-9aed-37bfb15d6be7.png)

Hope it helps you or someone else in any way.

**Comment by DeOldSax** [comment]
@freedome97 
Maybe the login steps changed (relevant file: https://github.com/DeOldSax/iliasDownloaderTool/blob/f0fda9a1e491986ea087daf6b9cec075f5f2ef65/src/main/java/plugin/KITIlias.java)

HTML Scraping for Dashboard courses: https://github.com/DeOldSax/iliasDownloaderTool/blob/f0fda9a1e491986ea087daf6b9cec075f5f2ef65/src/main/java/control/IliasScraper.java
